stumpy/aampi.py

```diff
@@ -103,7 +103,18 @@ def __init__(self, T, m, egress=True, p=2.0):
         )
 
         # Retrieve the left matrix profile values
-        for i, j in enumerate(self._left_I):
+
+        # Since each matrix profile value is the minimum between the left and right
+        # matrix profile values, we can save time by re-computing the left matrix
```

Comment:
It's subtle but I think it's better to say "re-computing ONLY the"

Reply:
sure.